env/io_posix.cc

```diff
@@ -709,7 +712,7 @@ PosixWritableFile::~PosixWritableFile() {
 Status PosixWritableFile::Append(const Slice& data) {
   assert(!use_direct_io() ||
          (IsSectorAligned(data.size(), GetRequiredBufferAlignment()) &&
-          IsPageAligned(data.data())));
+          IsPageAligned(data.data(), GetRequiredBufferAlignment())));
```

Comment:
Could you turn this into one assert per condition, so we can see better what fails?
Like
```
if (use_direct_io()) {
  assert(IsSectorAligned(data.size(), GetRequiredBufferAlignment());
  assert(IsPageAligned(data.data(), GetRequiredBufferAlignment());
}
```